packages/loopring_v3/contracts/impl/UserStakingPool.sol

```diff
@@ -169,13 +169,13 @@ contract UserStakingPool is Claimable, ReentrancyGuard, IUserStakingPool
 
             total.balance = total.balance.add(claimedAmount);
             total.claimedReward = total.claimedReward.add(claimedAmount);
-            total.claimedAt = (totalPoints >= userPoints) ?
-            now.sub(totalPoints.sub(userPoints) / total.balance) : now;
+            total.claimedAt = (uint64)((totalPoints >= userPoints) ?
```

Comment:
can we use `uint64(...)` instead of `(uint64)(...)`

Reply:
Done